app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict[str, str | int | None]]) -> list:
+    person_list = []
+
+    for human in people:
+        person_list.append(Person(human["name"], human["age"]))
+    for i, human in enumerate(people):
+        if "wife" in human and human["wife"] is not None:
```

Comment:
Use .get instead of two conditions the same with husband

Reply:
did you mean that?